logging/logging_release_notes/logging-5-8-release-notes.adoc

```diff
@@ -10,4 +10,4 @@ include::snippets/logging-compatibility-snip.adoc[]
 
 include::snippets/logging-stable-updates-snip.adoc[]
 
-// placeholder for 5.8 release notes
+include::modules/logging-release-notes-5-8-0.adoc[leveloffset=+1]
```

Comment:
I think you need to update the link in the `logging-compatibility-snip.adoc to the [OpenShift Operator Life Cycles -> Platform Agnostic Operators page](https://access.redhat.com/support/policy/updates/openshift_operators#platform-agnostic).